release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2463,20 +2463,49 @@ link:https://access.redhat.com/solutions/6164202[{product-title} 4.8.2 container
+
+Issued: 2021-08-09
+
+{product-title} release 4.8.4 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2021:2983[RHSA-2021:2983] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2021:2984[RHSA-2021:2984] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6246811[{product-title} 4.8.4 container image list]
+
+[id="ocp-4-8-4-bug-fixes"]
 ==== Bug fixes
 
+* Previously, link:https://bugzilla.redhat.com/show_bug.cgi?id=1954309[*BZ#1954309*] and link:https://bugzilla.redhat.com/show_bug.cgi?id=1960446[*BZ#1960446*] were listed as fixed bugs in the {product-title} 4.8.3 release notes, but were omitted from the version 4.8.3 release. With this release, the bug fix summaries are moved to the "Bug fixes" section of the {product-title} 4.8.4 release notes.
+
 * Previously, a bug in the lock implementation for the `nmstate-handler` pod caused multiple nodes to gain control. This update fixes the lock implementation so that only one node is in control of the lock. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954309[*BZ#1954309*])
 
 * Previously, there was an incorrect toleration setting on the `nmstate-handler` pod, which made network configuration on nodes with the `nmstate` Operator impossible. With this update, the handler pod allows toleration on all nodes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1960446[*BZ#1960446*])
 
-[id="ocp-4-8-3-upgrading"]
+* Previously, the web console showed `The operator is running in openshift-operators but is managing this namespace` for failed copied `ClusterServiceVersion` objects (CSVs). This message was not specific and did not help the user troubleshoot the failed CSV. With this release, the message for copied CSVs directs the user to the original CSV to find the cause of the failure, and provides a link to the original CSV. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1972478[*BZ#1972478*])
+
+* Previously, the Operator to check if the registry should use custom tolerations checked `spec.nodeSelector` instead of `spec.tolerations`, but the custom tolerations from `spec.tolerations` are applied only when `spec.nodeSelector` is set. With this release, the `spec.tolerations` is checked and the Operator uses custom tolerations if `spec.tolerations` is set. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1973662[*BZ#1973662*])
+
+* Previously, if a deployment was created with an unresolved image (meaning, without an image stream) and no `image.openshift.io/triggers` annotation, the deployment controller created replica sets in infinite loop. The issue has been resolved in this release. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1981770[*BZ#1981770*])
```

Comment:
Not critical, but I'd consider removing "meaning" from in the parentheses

Reply:
thanks! I actually reached out to Brandi about this one because it still seemed pretty off to me and she had a suggestion that I'm about to refresh this PR to include :)